stl/inc/numbers

```diff
@@ -0,0 +1,188 @@
+// numbers standard header (core)
+
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+#ifndef _NUMBERS_
+#define _NUMBERS_
+#include <yvals_core.h>
+#if _STL_COMPILER_PREPROCESSOR
+#if !_HAS_CXX20
+#pragma message("The contents of <numbers> are available only with C++20 or later.")
+#else // ^^^ !_HAS_CXX20 / _HAS_CXX20 vvv
+#ifdef __cpp_lib_concepts
+#include <concepts>
+#endif
+
+#pragma pack(push, _CRT_PACKING)
+#pragma warning(push, _STL_WARNING_LEVEL)
+#pragma warning(disable : _STL_DISABLED_WARNINGS)
+_STL_DISABLE_CLANG_WARNINGS
+#pragma push_macro("new")
+#undef new
+
+_STD_BEGIN
+namespace numbers {
+    template <class _Ty>
+    struct _Invalid {
+        static_assert(_Always_false<_Ty>, "This variable template is not specialised for this type which is invalid.");

```

Comment:
First, I recommend citing N4835 [math.constants]/3.

Second, this should say "specialized"; like the Standard, we use the en-us spelling in source code.

Third, there is potential ambiguity in the phrasing. You're trying to say that the lack of a specialization is invalid, but it sounds like the type is invalid.

To solve these issues, I'd recommend just quoting the Standardese directly, which seems sufficiently clear: "A program that instantiates a primary template of a mathematical constant variable template is ill-formed. (N4835 [math.constants]/3)"